client/src/main/java/io/pravega/client/control/impl/ControllerResolverFactory.java

```diff
@@ -133,6 +133,7 @@ public String getDefaultScheme() {
          * @param enableDiscovery   Whether to use the controller's discovery API.
          * @param executor          The executor to run resolve tasks on.
          */
+        @SuppressWarnings("deprecation")
```

Comment:
which api that we use is deprecated? shall we update the code to use the replacement api?

Reply:
ManagedChannelBuilder#nameResolverFactory() becomes marked as deprecated

elsewhere, JsonParser#parse() also become marked as deprecated